tool/microkit/src/main.rs

```diff
@@ -1801,7 +1801,8 @@ fn build_system(kernel_config: &Config,
 
     for (i, pd) in system.protection_domains.iter().enumerate() {
         let elf = &mut pd_elf_files[i];
-        elf.write_symbol("microkit_name", pd.name.as_bytes())?;
+        let pd_name_bytes = pd.name.as_bytes();
+        elf.write_symbol("microkit_name", &pd_name_bytes[..pd_name_bytes.len().min(PD_MAX_NAME_LENGTH)])?;
```

Comment:
Can you do `let name_length = min(pd_name_bytes, PD_MAX_NAME_LENGTH);` like below to make it a bit more readable.

Reply:
Done